sycl/include/sycl/detail/pi.h

```diff
@@ -1925,6 +1927,55 @@ pi_result piextEnqueueDeviceGlobalVariableRead(
 ///
 /// Plugin
 ///
+///
+// Host Pipes
+///
+
+/// Read from pipe of a given name
+///
+/// @param queue a valid host command-queue in which the read / write command
+/// will be queued. command_queue and program must be created with the same
+/// OpenCL context.
+/// @param program a program object with a successfully built executable.
+/// @param pipe_symbol the name of the program scope pipe global variable.
+/// @param blocking indicate if the read and write operations are blocking or
+/// non-blocking
+/// @param ptr a pointer to buffer in host memory that will hold resulting data
+/// from pipe
+/// @param size size of the memory region to read or write, in bytes.
+/// @param num_events_in_waitlist number of events in the wait list.
+/// @param events_waitlist specify events that need to complete before this
+/// particular command can be executed.
+/// @param event returns an event object that identifies this read / write
+/// command and can be used to query or queue a wait for this command to
+/// complete.
+__SYCL_EXPORT pi_result piextEnqueueReadHostPipe(
```

Comment:
@kbenzie: please take this change to Unified Runtime

Reply:
@zibaiwan : please update PI API version here https://github.com/intel/llvm/pull/7468/files#diff-952d2515bc7a8c91d37420f519ff726c0e6ce71f5a81bfd98e5c6fcfcaa3ada0R78-R82

Reply:
@smaslov-intel , thank you very much for your review. I have update the PI version in the pi.h. Sorry I missed this earlier. Please take a look!

Reply:
@smaslov-intel tracking this in https://github.com/oneapi-src/unified-runtime/issues/400